packages/jwt-verifier/test/integration-test/resource-server.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
```

Comment:
we should run `npm run banners` in the root of `okta-oidc-js` to ensure all of our js has a license banner

Reply:
Done